src/rules/converters/no-import-side-effect.ts

```diff
@@ -0,0 +1,21 @@
+import { RuleConverter } from "../converter";
+
+export const convertNoImportSideEffect: RuleConverter = tsLintRule => {
+    const rules = [];
+    const notices = [];
+
+    if (tsLintRule.ruleArguments.length > 0) {
+        rules.push({ allow: tsLintRule.ruleArguments[1]["ignore-module"] });
+        notices.push("This now accepts a glob pattern, so your regex may need converted");
```

Comment:
What do you think of slight phrasing change?

```suggestion
        notices.push("ESLint's no-import-side-effect now accepts a glob pattern for ignores; you'll need to manually convert your ignore-module settings.");
```